app/views/admin/admin/_admintab.html.erb

```diff
@@ -0,0 +1,26 @@
+<link href="//maxcdn.bootstrapcdn.com/bootstrap/3.3.0/css/bootstrap.min.css" rel="stylesheet" id="bootstrap-css">
+<script src="//maxcdn.bootstrapcdn.com/bootstrap/3.3.0/js/bootstrap.min.js"></script>
```

Comment:
you already have bootstrap in your application, remove these lines.

Reply:
Have removed the file.